src/content/docs/tutorial-errors/respond-outages.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Respond to service outages
+metaDescription: Learn how to monitor your system so you can quickly identify and resolve many error occurrences fast. 
+---
+
+import allEntitiesView from 'images/apm_screenshot-crop_all_entities.webp'
+
+import apiGateway from 'images/apm_screenshot-crop_api-gateway-summary.webp'
+
+import apmErrorsInboxPage from 'images/apm_screenshot-crop_errors-inbox-page.webp'
+
+import apmErrorsAnomaly from 'images/apm_screenshot-crop_errors-anomaly.webp'
+
+Errors are bound to happen. Even with an observability tool, finding the source of an error isn't as straightforward as you might assume. Think about a flooded yard: you notice water flowing near your hose bib, but the cause of the flood is actually a crack somewhere in your water main. If you assumed that the leaking hose caused the flood, you'd end up with a fixed hose but a ruined lawn. The mistake would be costly. 
```

Comment:
I like this example but I feel like hose bib might not be as readily understood as a real life example might expect to be. Maybe just removing "bib" will work? Just a thought.

Reply:
I think the last sentence should be joined to the previous. "fixed hose but a ruined lawn: a costly mistake."

Reply:
Ok, ok You and @homelessbirds both pointed this out so I will change it 😭

Reply:
(the bib refers to the place where the water comes out, whereas the hose itself is the long thing you attach to the bib)